interface/patient_file/summary/demographics.php

```diff
@@ -1199,11 +1202,19 @@ function filterActiveIssues(array $i): array
                     endif;
 
                     $sectionRenderEvents = $ed->dispatch(new SectionEvent('primary'), SectionEvent::EVENT_HANDLE);
+                    $sectionRenderEvents->addCard(new DemographicsViewCard($result, $result2, ['dispatcher' => $ed]));
```

Comment:
@robertdown You may want to look at what I did here with pulling these codes into individual cards that eventually could be resorted/rearranged.  Hopefully it's flowing the way it should.